src/plugins/controls/public/options_list/embeddable/options_list_embeddable.tsx

```diff
@@ -425,11 +445,16 @@ export class OptionsListEmbeddable extends Embeddable<OptionsListEmbeddableInput
     const { Wrapper: OptionsListReduxWrapper } = this.reduxEmbeddableTools;
     this.node = node;
     ReactDOM.render(
-      <KibanaThemeProvider theme$={pluginServices.getServices().theme.theme$}>
-        <OptionsListReduxWrapper>
-          <OptionsListControl typeaheadSubject={this.typeaheadSubject} />
-        </OptionsListReduxWrapper>
-      </KibanaThemeProvider>,
+      <I18nProvider>
```

Comment:
The `FormattedMessage` component for the `search.allow_expensive_queries` warning in the `popover_title` was getting mad that there was no `I18n` provider unfortunately 🙃